managers/coverFetchers/publisher_backlist_cover_fetcher.py

```diff
@@ -0,0 +1,80 @@
+import hashlib
+import io
+import os
+
+import PIL
+import pypdf
+import requests
+import requests.exceptions
+
+from logger import create_log
+from managers.coverFetchers.abstractFetcher import AbstractFetcher
+
+logger = create_log(__name__)
+
+
+class PublisherBacklistCoverFetcher(AbstractFetcher):
```

Comment:
Could this be just the PDFCoverFetcher? Understand we do have some hardcoding here for the Schomburg files.